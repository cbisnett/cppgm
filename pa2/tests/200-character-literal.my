literal 'a' char 61
literal 'π' int C0030000
literal '𝄞' int 1ED10100
literal '\'' char 27
literal '\"' char 22
literal '\?' char 3F
literal '\\' char 5C
literal '\a' char 07
literal '\b' char 08
literal '\f' char 0C
literal '\n' char 0A
literal '\r' char 0D
literal '\t' char 09
literal '\v' char 0B
literal 'π' int C0030000
literal '𝄞' int 1ED10100
literal '\x3C0' int C0030000
literal '\x1D11E' int 1ED10100
literal '\101' char 41
literal u'a' char16_t 6100
literal u'π' char16_t C003
invalid u'𝄞'
literal u'\'' char16_t 2700
literal u'\"' char16_t 2200
literal u'\?' char16_t 3F00
literal u'\\' char16_t 5C00
literal u'\a' char16_t 0700
literal u'\b' char16_t 0800
literal u'\f' char16_t 0C00
literal u'\n' char16_t 0A00
literal u'\r' char16_t 0D00
literal u'\t' char16_t 0900
literal u'\v' char16_t 0B00
literal u'π' char16_t C003
invalid u'𝄞'
literal u'\x3C0' char16_t C003
invalid u'\x1D11E'
literal u'\101' char16_t 4100
literal U'a' char32_t 61000000
literal U'π' char32_t C0030000
literal U'𝄞' char32_t 1ED10100
literal U'\'' char32_t 27000000
literal U'\"' char32_t 22000000
literal U'\?' char32_t 3F000000
literal U'\\' char32_t 5C000000
literal U'\a' char32_t 07000000
literal U'\b' char32_t 08000000
literal U'\f' char32_t 0C000000
literal U'\n' char32_t 0A000000
literal U'\r' char32_t 0D000000
literal U'\t' char32_t 09000000
literal U'\v' char32_t 0B000000
literal U'π' char32_t C0030000
literal U'𝄞' char32_t 1ED10100
literal U'\x3C0' char32_t C0030000
literal U'\x1D11E' char32_t 1ED10100
literal U'\101' char32_t 41000000
literal L'a' wchar_t 61000000
literal L'π' wchar_t C0030000
literal L'𝄞' wchar_t 1ED10100
literal L'\'' wchar_t 27000000
literal L'\"' wchar_t 22000000
literal L'\?' wchar_t 3F000000
literal L'\\' wchar_t 5C000000
literal L'\a' wchar_t 07000000
literal L'\b' wchar_t 08000000
literal L'\f' wchar_t 0C000000
literal L'\n' wchar_t 0A000000
literal L'\r' wchar_t 0D000000
literal L'\t' wchar_t 09000000
literal L'\v' wchar_t 0B000000
literal L'π' wchar_t C0030000
literal L'𝄞' wchar_t 1ED10100
literal L'\x3C0' wchar_t C0030000
literal L'\x1D11E' wchar_t 1ED10100
literal L'\101' wchar_t 41000000
invalid 'abc'
eof
