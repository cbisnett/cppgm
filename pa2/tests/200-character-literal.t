'a' 'π' '𝄞' '\'' '\"' '\?' '\\' '\a' '\b' '\f' '\n' '\r' '\t' '\v'

'\u03C0' '\U0001D11E' '\x3C0' '\x1D11E' '\101'

u'a' u'π' u'𝄞' u'\'' u'\"' u'\?' u'\\' u'\a' u'\b' u'\f' u'\n' u'\r' u'\t' u'\v'

u'\u03C0' u'\U0001D11E' u'\x3C0' u'\x1D11E' u'\101'

U'a' U'π' U'𝄞' U'\'' U'\"' U'\?' U'\\' U'\a' U'\b' U'\f' U'\n' U'\r' U'\t' U'\v'

U'\u03C0' U'\U0001D11E' U'\x3C0' U'\x1D11E' U'\101'

L'a' L'π' L'𝄞' L'\'' L'\"' L'\?' L'\\' L'\a' L'\b' L'\f' L'\n' L'\r' L'\t' L'\v'

L'\u03C0' L'\U0001D11E' L'\x3C0' L'\x1D11E' L'\101'

'abc'

