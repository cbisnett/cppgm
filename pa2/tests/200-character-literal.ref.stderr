ERROR: UTF-16 char literal out of range: u'𝄞' 119070
ERROR: UTF-16 char literal out of range: u'𝄞' 119070
ERROR: UTF-16 char literal out of range: u'\x1D11E' 119070
ERROR: multi code point character literals not supported: 'abc'
